### PR TITLE
Internal variable renamed for consistency with next updates

### PR DESCRIPTION
**Pull request by mvaligursky**
just a rename

### CODE REVIEW

File: examples/src/examples/graphics/ground-fog.tsx [hunk 1]
```diff
@@ -256,8 +256,9 @@ class GroundFogExample {
 
                 let firstFrame = true;
                 let currentTime = 0;
+                let c = 0;
                 app.on("update", function (dt) {
-
+                    c++;
```
Comment (willeastcott) [inline examples/src/examples/graphics/ground-fog.tsx:261]: I did a double-take here. C++? This is JavaScript!

File: examples/src/examples/graphics/ground-fog.tsx [hunk 2]
```diff
@@ -274,6 +275,11 @@ class GroundFogExample {
 
                     // debug rendering of the deptht texture in the corner
                     app.drawDepthTexture(0.7, -0.7, 0.5, -0.5);
+
+                    pc.Tracing.set(pc.TRACEID_RENDER_PASS, c === 20);
+                    pc.Tracing.set(pc.TRACEID_RENDER_PASS_DETAIL, c === 20);
+
+
```
Comment (willeastcott) [inline examples/src/examples/graphics/ground-fog.tsx:282]: What's with all the whitespace? 😄

File: examples/src/examples/graphics/ground-fog.tsx (hunk 1 above)
Comment (mvaligursky) [reply examples/src/examples/graphics/ground-fog.tsx:261]: right? sorry removing changes to this file, c++ just won't do it.

File: examples/src/examples/graphics/ground-fog.tsx [hunk 3]
```diff
@@ -274,6 +275,11 @@ class GroundFogExample {
 
                     // debug rendering of the deptht texture in the corner
                     app.drawDepthTexture(0.7, -0.7, 0.5, -0.5);
+
+                    pc.Tracing.set(pc.TRACEID_RENDER_PASS, c === 20);
+                    pc.Tracing.set(pc.TRACEID_RENDER_PASS_DETAIL, c === 20);
```
Comment (willeastcott) [inline examples/src/examples/graphics/ground-fog.tsx]: Feels like you didn't mean to include changes to this example in your PR?

File: src/scene/graphics/render-pass-depth.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import { RenderPass } from "../../platform/graphics/render-pass.js";
```
Comment (willeastcott) [inline src/scene/graphics/render-pass-depth.js:1]: Is this meant to be in the PR?

Comment (mvaligursky) [reply src/scene/graphics/render-pass-depth.js:1]: nope :(
too keen to get that rename going. This is next PR, sorry.

**Pull request merged**